File: packages/insomnia-httpsnippet/test/fixtures/output/csharp/restsharp/multipart-data.cs
var client = new RestClient("http://mockbin.com/har");
var request = new RestRequest(Method.POST);
request.AddHeader("content-type", "multipart/form-data; boundary=---011000010111000001101001");
request.AddParameter("multipart/form-data; boundary=---011000010111000001101001", "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"; filename=\"hello.txt\"\r\nContent-Type: text/plain\r\n\r\nHello World\r\n-----011000010111000001101001--\r\n", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);

File: packages/insomnia-httpsnippet/test/fixtures/output/node/request/multipart-data.js
var request = require("request");

var options = { method: 'POST',
  url: 'http://mockbin.com/har',
  headers: { 'content-type': 'multipart/form-data; boundary=---011000010111000001101001' },
  formData: 
   { foo: 
      { value: 'Hello World',
        options: { filename: 'hello.txt', contentType: 'text/plain' } } } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

